jobclient/python/cook/scheduler/client/__init__.py

```diff
@@ -12,25 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import getpass
 import json
 import logging
 
 import requests
 
-import util
-
-from typing import Dict, Optional
+from datetime import timedelta
+from typing import Dict, Optional, Union
 from urllib.parse import urlencode, urlunparse
 from uuid import UUID
 
-from jobs import Application, Job
+from . import util
+from .jobs import Application, Job
 
 _LOG = logging.getLogger(__name__)
 _LOG.addHandler(logging.StreamHandler())
 _LOG.setLevel(logging.DEBUG)
 
 _COOK_IMPERSONATE_HEADER = 'X-Cook-Impersonate'
 
+_CLIENT_APP = Application('cook-python-client', '0.1')
```

Comment:
Let's use semantic versioning (https://semver.org/)

Reply:
What version should we start with? `0.1.0`?

Reply:
Yeah, `0.1.0` sounds good